resources/Microsoft.Windows.Setting.Language/Microsoft.Windows.Setting.Language.psm1

```diff
@@ -0,0 +1,213 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+using namespace System.Collections.Generic
+
+if ([string]::IsNullOrEmpty($env:TestRegistryPath)) {
+    $global:LocaleNameRegistryPath = 'HKCU:\Control Panel\International'
+    $global:LocaleUserProfilePath = 'HKCU:\Control Panel\International\User Profile'
+} else {
+    $global:LocaleNameRegistryPath = $global:LocaleUserProfilePath = $env:TestRegistryPath
+}
+
+#region Functions
+function Get-OsBuildVersion {
+    return [System.Environment]::OSVersion.Version.Build
+}
```

Comment:
I would just call this function directly instead of creating another one around it.